thefuck/rules/omnienv_no_such_command.py

```diff
@@ -1,33 +1,34 @@
 import re
-from subprocess import PIPE, Popen
-
 from thefuck.utils import (cache, for_app, replace_argument, replace_command,
                            which)
+from subprocess import PIPE, Popen
+
+
+supported_apps = 'goenv', 'nodenv', 'pyenv', 'rbenv'
```

Comment:
We can add `jenv` to the list too.

```suggestion
supported_apps = 'goenv', 'nodenv', 'pyenv', 'rbenv', 'jenv'
```

Reply:
From what I could check, `jenv` does not follow the same command/output pattern.

Reply:
Ohh, okay. I saw similar commands being used there, and they mentioned that it is derived from `rbenv` so I thought it would be compatible too.

Reply:
Yeah, that's a shame. I wish it was compatible.

BTW, if it wasn't for @gkodosis — thanks, man! — I'd prolly never tell how almost identical those four are. And I use `pyenv` daily and have used both `goenv` and `rbenv` a couple years ago. 😄